src/transaction/log_page_buffer.c

```diff
@@ -3858,6 +3858,7 @@ logpb_flush_all_append_pages (THREAD_ENTRY * thread_p)
   LOG_FLUSH_INFO *flush_info = &log_Gl.flush_info;
   LOGWR_INFO *writer_info = &log_Gl.writer_info;
   LOG_PAGE *first_append_log_page = NULL;
+  LOG_PAGE *copy_to_first_append = NULL;
```

Comment:
@funkydvd 

What about using static buffer for `copy_to_first_append`?

Please refer to code snippet as follows:
```C
  char log_pgbuf[IO_MAX_PAGE_SIZE + MAX_ALIGNMENT];
  LOG_PAGE *log_page_p = NULL;

  ...

  log_page_p = (LOG_PAGE *) PTR_ALIGN (log_pgbuf, MAX_ALIGNMENT);
```